test/system/user_index_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :feature do
+  describe 'GET /users' do
+    let!(:user1) { create(:user, name: 'Tom', postscounter: 3) }
+    let!(:user2) { create(:user, name: 'sasas', postscounter: 3) }
+    before do
+      visit users_path
+    end
+    it 'displays all users' do
+      expect(page).to have_content('List of All Users')
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20the%20number%20of%20posts%20each%20user%20has%20written.), Please add a test that asserts you can see the number of posts each user has written.

Reply:
expect(page).to have_content("Total number of posts: #{user1.postscounter}")

Added the missing test.